plugins/inputs/kinesis_consumer/kinesis_consumer.go

```diff
@@ -89,6 +91,25 @@ func (k *KinesisConsumer) SetParser(parser telegraf.Parser) {
 	k.parser = parser
 }
 
+type TelegrafLoggerWrapper struct {
+	telegraf.Logger
+}
+
+func (t *TelegrafLoggerWrapper) Log(args ...interface{}) {
+	t.Debug(args...)
+}
+
+func (t *TelegrafLoggerWrapper) Logf(classification logging.Classification, format string, v ...interface{}) {
```

Comment:
How about making the two functions members of `KinesisConsumer`? This way you don't need to pass around the logger.

Reply:
sorry, would you clarify a bit pls, not sure I fully understand what you mean.
If you suggest changing the receiver of these 2 functions to `k *KinesisConsumer` it already has Log field so it will be a name clash.

Reply:
Ok I see. So keep it the way you have it.